### DISCUSSION TITLE
编译DLL错误: 类型“Object”在未引用的程序集中定义。必须添加对程序集“System.Runtime

### DISCUSSION THREAD

**Discussion by xibaoning** [dotnetcore/Natasha#218] [Q&A]
类型“Object”在未引用的程序集中定义。必须添加对程序集“System.Runtime, Version=7.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a”的引用。

已经使用NatashaManagement.AddGlobalReference(typeof(Object));

**Comment by NMSAzulX** [comment]
这个问题很复杂，官方的 API 默认寻找的是 dll 的位置，官方不支持从已加载的类型中提取引用。不过 Natasha 会在下个版本中解决。

**Reply by NMSAzulX** [reply]
已放入 ISSUE 中。
